lib/appium-driver.d.ts

```diff
@@ -164,18 +162,26 @@ export declare class AppiumDriver {
      * @param inertia
      * @param xOffset
      */
-    swipe(y: number, x: number, yOffset: number, inertia?: number, xOffset?: number): Promise<void>;
+    swipe(startPoint: {
```

Comment:
Can we change the documentation parameters + to change the order we provide coordinates to be x,y as most places start with x. Here is vice versa y,x which is not intuitive

Reply:
Yep. although this is an object and simply properties which doesn't reflect of the order of passing args.